.github/workflows/ci.yml

```diff
@@ -49,6 +45,10 @@ concurrency:
 jobs:
   setup:
     uses: ./.github/workflows/setup.yml
+    secrets: inherit
```

Comment:
Do we have any secret beside the GitHub token and is this needed if only the GitHub token is used?